src/shared/utils.js

```diff
@@ -1,6 +1,6 @@
 function convertToSafeString(string, seperator="_"){
-	return string.replace(/\W+/g, '') // remove non-alphanumeric
-			.replace(/ /g, seperator)      // replace spaces with underscores or provided seperator
+
+	return string.replace(/\W+/g, seperator) // remove non-alphanumeric & space, inset seperators
 			.toLowerCase();
 }//convertToSafeString
```

Comment:
is this a duplicate of the change you made in the other branch? heads-up whenever you do something like that! If they're exactly the same then you should be able to merge them without any issues, but this is how things tend to fall out of sync and cause problems. If you want to expand on the work done from another branch you can always checkout a new branch _from that branch_ instead of `develop` (or whatever our base is)

Reply:
Not a duplicate, it was a bug fix to the change I made on the episode-preview branch. Having the two replace functions was doing something weird and resulting in strings with no spaces and no seperators.